soh/soh/Enhancements/randomizer/location_list.cpp

```diff
@@ -1036,6 +1045,557 @@ void Rando::StaticData::InitLocationTable() { //
+	locationTable[RC_WATER_TEMPLE_MQ_LOWEST_GS_POT_2] 			    = Location::Pot(RC_WATER_TEMPLE_MQ_LOWEST_GS_POT_2, 			RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		SCENE_WATER_TEMPLE, 		TWO_ACTOR_PARAMS(-2314, 808), 	"MQ Lowest GS Pot 2", 			"Water Temple MQ Lowest GS Pot 2", 		    	    RHT_POT_WATER_TEMPLE,	        RG_BOMBS_5,         SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_LOWEST_GS_POT_2));
+	locationTable[RC_WATER_TEMPLE_MQ_LOWEST_GS_POT_3] 			    = Location::Pot(RC_WATER_TEMPLE_MQ_LOWEST_GS_POT_3, 			RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		SCENE_WATER_TEMPLE, 		TWO_ACTOR_PARAMS(-2313, 944), 	"MQ Lowest GS Pot 3", 			"Water Temple MQ Lowest GS Pot 3", 		    	    RHT_POT_WATER_TEMPLE,	        RG_ARROWS_10,       SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_LOWEST_GS_POT_3));
+	locationTable[RC_WATER_TEMPLE_MQ_LOWEST_GS_POT_4] 			    = Location::Pot(RC_WATER_TEMPLE_MQ_LOWEST_GS_POT_4, 			RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		SCENE_WATER_TEMPLE, 		TWO_ACTOR_PARAMS(-2313, 905), 	"MQ Lowest GS Pot 4", 			"Water Temple MQ Lowest GS Pot 4", 		    	    RHT_POT_WATER_TEMPLE,	        RG_ARROWS_10,       SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_LOWEST_GS_POT_4));
+	locationTable[RC_WATER_TEMPLE_MQ_STORAGE_ROOM_B_POT_1] 		    = Location::Pot(RC_WATER_TEMPLE_MQ_STORAGE_ROOM_B_POT_1, 		RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		SCENE_WATER_TEMPLE, 		TWO_ACTOR_PARAMS(-1417, -3025), "MQ Storage Room B Pot 1", 		"Water Temple MQ Storage Room B Pot 1", 	        RHT_POT_WATER_TEMPLE,	        RG_GREEN_RUPEE,     SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_STORAGE_ROOM_B_POT_1));
+	locationTable[RC_WATER_TEMPLE_MQ_STORAGE_ROOM_B_POT_2] 	    	= Location::Pot(RC_WATER_TEMPLE_MQ_STORAGE_ROOM_B_POT_2, 		RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		SCENE_WATER_TEMPLE, 		TWO_ACTOR_PARAMS(-1150, -3031), "MQ Storage Room B Pot 2", 		"Water Temple MQ Storage Room B Pot 2",     	    RHT_POT_WATER_TEMPLE,	        RG_BLUE_RUPEE,      SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_STORAGE_ROOM_B_POT_2));
+	locationTable[RC_WATER_TEMPLE_MQ_MINI_DODONGO_POT_1] 	    	= Location::Pot(RC_WATER_TEMPLE_MQ_MINI_DODONGO_POT_1, 		    RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		SCENE_WATER_TEMPLE, 		TWO_ACTOR_PARAMS(28, -3817), 	"MQ Mini Dodongo Pot 1", 		"Water Temple MQ Mini Dodongo Pot 1", 	    	    RHT_POT_WATER_TEMPLE,	        RG_BOMBS_5,         SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_MINI_DODONGO_POT_1));
+	locationTable[RC_WATER_TEMPLE_MQ_MINI_DODONGO_POT_2] 	    	= Location::Pot(RC_WATER_TEMPLE_MQ_MINI_DODONGO_POT_2, 		    RCQUEST_MQ, RCAREA_WATER_TEMPLE, 		SCENE_WATER_TEMPLE, 		TWO_ACTOR_PARAMS(-28, -3817), 	"MQ Mini Dodongo Pot 2", 		"Water Temple MQ Mini Dodongo Pot 2", 	    	    RHT_POT_WATER_TEMPLE,	        RG_BOMBS_5,         SpoilerCollectionCheck::RandomizerInf(RAND_INF_WATER_TEMPLE_MQ_MINI_DODONGO_POT_2));
+	
```

Comment:
You forgot to add the GTG pots, there's at least 4 in MQ, maybe more will search